src/trix/trix.js

```diff
@@ -23,6 +23,11 @@ const Trix = {
   filters
 }
 
+Trix.start = () => {
```

Comment:
Is there a reason to define `start` outside the `const Trix { ... }` literal above it? You could do it inline, or define it as `start` first and include it along with the others.

I don't think it matters too much either way. I was just curious about the stylistic choice.

Reply:
No reason, I hadn't realised it's a bit weird. Changed in 5f598cc3dd8849fcc3393a60d81631848370e7e1.